pliers/export.py

```diff
@@ -1,70 +0,0 @@
-from os.path import exists, isdir, join
-from abc import ABCMeta, abstractmethod
-import pandas as pd
-from six import with_metaclass
-from pliers.extractors.base import ExtractorResult
-
-
-class Exporter(with_metaclass(ABCMeta)):
-
-    ''' Base exporter class. '''
-
-    @abstractmethod
-    def export(self):
-        pass
-
-
-def to_long_format(df):
```

Comment:
I found this method/snippet useful for the Merlin analysis. Could we move it to `utils` or something?